tools/bigquery-zos-mainframe-connector/USING_WITH_BQ.md

```diff
@@ -0,0 +1,227 @@
+# Using with BigQuery
+
+## bq query
+
```

Comment:
Add notes about the various ways queries can be provided - inline, separate dataset via DD, separate dataset via DSN.
Also add notes about how users may want to apply their own parameterization tokens to the query datasets separately from what is provided by the connector. This means that they would need a preceding job step to replace parameter tokens to produce the query dataset.

Reply:
added